Compilation on src/util.c breaks on macos Ventura 13.0.1

I get this error:

```
FAILED: src/transmission-remote-gtk.p/util.c.o 
cc -Isrc/transmission-remote-gtk.p -Isrc -I../src -I/usr/local/Cellar/gtk+3/3.24.34/include/gtk-3.0 -I/usr/local/Cellar/glib/2.74.3/include/gio-unix-2.0 -I/usr/local/Cellar/cairo/1.16.0_5/include/cairo -I/usr/local/Cellar/libepoxy/1.5.10/include -I/usr/local/Cellar/pango/1.50.12/include/pango-1.0 -I/usr/local/Cellar/harfbuzz/5.3.1/include/harfbuzz -I/usr/local/Cellar/fribidi/1.0.12/include/fribidi -I/usr/local/Cellar/graphite2/1.3.14/include -I/usr/local/Cellar/atk/2.38.0/include/atk-1.0 -I/usr/local/Cellar/pixman/0.40.0/include/pixman-1 -I/usr/local/Cellar/fontconfig/2.14.1/include -I/usr/local/opt/freetype/include/freetype2 -I/usr/local/Cellar/libxcb/1.15/include -I/usr/local/Cellar/libxrender/0.9.10/include -I/usr/local/Cellar/libxext/1.3.5/include -I/usr/local/Cellar/libx11/1.8.2/include -I/usr/local/Cellar/libxau/1.0.10/include -I/usr/local/Cellar/libxdmcp/1.1.3/include -I/usr/local/Cellar/gdk-pixbuf/2.42.10/include/gdk-pixbuf-2.0 -I/usr/local/Cellar/libpng/1.6.39/include/libpng16 -I/usr/local/Cellar/jpeg-turbo/2.1.4/include -I/usr/local/Cellar/libtiff/4.4.0_1/include -I/usr/local/Cellar/glib/2.74.3/include -I/usr/local/Cellar/glib/2.74.3/include/glib-2.0 -I/usr/local/Cellar/glib/2.74.3/lib/glib-2.0/include -I/usr/local/opt/gettext/include -I/usr/local/Cellar/pcre2/10.40/include -I/usr/local/Cellar/xorgproto/2022.2/include -I/Library/Developer/CommandLineTools/SDKs/MacOSX10.15.sdk/usr/include/ffi -I/usr/local/Cellar/json-glib/1.6.6/include/json-glib-1.0 -I/usr/local/Cellar/icu4c/71.1/include -I/usr/local/Cellar/libsoup/3.2.2/include/libsoup-3.0 -I/usr/local/Cellar/libpsl/0.21.1_5/include -I/usr/local/Cellar/libnghttp2/1.51.0/include -I/usr/local/Cellar/geoip/1.6.12/include/ -fcolor-diagnostics -Wall -Winvalid-pch -Wextra -std=gnu99 -O2 -g -DGLIB_VERSION_MIN_REQUIRED=GLIB_VERSION_2_56 -DGLIB_VERSION_MAX_ALLOWED=GLIB_VERSION_2_70 -funsigned-char -fstack-protector-strong -Winline -Wno-unused-parameter -Wstrict-prototypes -Wmissing-prototypes -Werror=format=2 -Werror=implicit-function-declaration -Werror=pointer-arith -Werror=init-self -Werror=missing-include-dirs -Werror=date-time -Wdeprecated-declarations -Wdeprecated -fPIE -MD -MQ src/transmission-remote-gtk.p/util.c.o -MF src/transmission-remote-gtk.p/util.c.o.d -o src/transmission-remote-gtk.p/util.c.o -c ../src/util.c
../src/util.c:332:9: error: expected expression
        const gchar *resultStr = json_object_get_string_member(response, "result");
        ^
../src/util.c:333:25: error: use of undeclared identifier 'resultStr'
        return g_strdup(resultStr ? resultStr : "Server responded, but with no result.");
                        ^
../src/util.c:333:37: error: use of undeclared identifier 'resultStr'
        return g_strdup(resultStr ? resultStr : "Server responded, but with no result.");
                                    ^
3 errors generated.
ninja: build stopped: subcommand failed.

this is the offending code:

case FAIL_RESULT_UNSUCCESSFUL: 
        const gchar *resultStr = json_object_get_string_member(response, "result");
        return g_strdup(resultStr ? resultStr : "Server responded, but with no result.");

putting braces to it fixed the problem for me

case FAIL_RESULT_UNSUCCESSFUL: {
        const gchar *resultStr = json_object_get_string_member(response, "result");
        return g_strdup(resultStr ? resultStr : "Server responded, but with no result.");
     }
```

--- a/src/util.c
+++ b/src/util.c
@@ -328,9 +328,10 @@ gchar *make_error_message(JsonObject *response, gint status, gchar *err_msg)
     case FAIL_JSON_DECODE:
         return g_strdup(err_msg ? err_msg : "Unknown JSON decoding error.");
 
-    case FAIL_RESULT_UNSUCCESSFUL:
+    case FAIL_RESULT_UNSUCCESSFUL: {
         const gchar *resultStr = json_object_get_string_member(response, "result");
         return g_strdup(resultStr ? resultStr : "Server responded, but with no result.");
+    }
 
     case FAIL_NO_SESSION_ID:
         return g_strdup("No \"" TRANSMISSION_SESSION_ID_HEADER

